### PR TITLE
ref(Dockerfile): ditch the pip cache for a slightly smaller image

### PR DESCRIPTION
**Pull request by helgi**

### CODE REVIEW

File: rootfs/Dockerfile [hunk 1]
```diff
@@ -10,19 +10,20 @@ RUN adduser --system \
...
     ln -s /usr/bin/python3 /usr/bin/python && \
-	curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.2 && \
-	mkdir -p /configs && chown -R deis:deis /configs && \
-	pip install --disable-pip-version-check --no-cache-dir -r /app/requirements.txt && \
-	apt-get purge -y --auto-remove $buildDeps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc
+    curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.2 && \
+    mkdir -p /configs && chown -R deis:deis /configs && \
+    pip install --disable-pip-version-check --no-cache-dir -r /app/requirements.txt && \
```
Comment (Joshua-Anderson) [inline rootfs/Dockerfile]: Doesn't `--no-cache-dir` already do this?

Comment (helgi) [inline rootfs/Dockerfile]: You'd think but if you drop onto the controller container and `dh -h` (or similar) that dir, it has data. Can you try it on yours just to confirm?

**Comment by helgi** [comment]
Also considering removing pip itself

**Comment by Joshua-Anderson** [comment]
Removing pip makes sense. Any reason not to?

**Comment by helgi** [comment]
Mostly if we are extending main to dev would be a reason to keep it but that's about it!

**Comment by Joshua-Anderson** [comment]
LGTM, but we should probably investigate why this is neccessary (maybe open an issue?). Could it be that `--no-cache` only partially disables caching?

**Pull request merged**